scripts/setup/create_multinode.sh

```diff
@@ -58,8 +58,13 @@ server_exec() {
 
 common_init() {
     internal_init() {
-        server_exec $1 "git clone --branch=$VHIVE_BRANCH https://github.com/ease-lab/vhive"
-        server_exec $1 "cd; ./vhive/scripts/cloudlab/setup_node.sh $OPERATION_MODE"
+        server_exec $1 "git clone --branch=$VHIVE_BRANCH https://github.com/wanghanchengchn/vHive.git vhive"
+
+        server_exec $1 'sudo wget -q https://github.com/mikefarah/yq/releases/download/v4.27.3/yq_linux_amd64 -O /usr/bin/yq && sudo chmod +x /usr/bin/yq'
+        server_exec $1 'bash -s' < ~/invitro/scripts/setup/rewrite_yaml_files.sh
```

Comment:
Can this be done after cloning the invitro to master node? So it would be just running script instead of passing it through ssh?